_posts/AzerothCoreHostedTutorial.md

```diff
@@ -0,0 +1,144 @@
+## Introduction
+This tutorial will walk you through the entire AzerothCore server setup process, and will help get you familiar with using Debian and Command Line Interfaces (CLI). In this tutorial, we'll be going over the following:
+- Looking at various tools to help you get your server up and running, such as MySQL Clients, SSH Clients, SFTP Clients, and powerful file editors. 
+- Configuring a Droplet on DigitalOcean from scratch
+- Building the Core
+- Loading the Database
+
+There is a Step-by-Step YouTube Video which walks you through this entire process. It's encouraged you follow along with that video, while using this tutorial as a reference guide.
+
+[![Tutorial Video](https://img.youtube.com/vi/lenMiDtbHhs/0.jpg)](https://www.youtube.com/watch?v=lenMiDtbHhs)
+
+## PC Apps You May Need:
+#### MySQL Clients:
+- [HeidiSQL](https://www.heidisql.com/)
+- [MySQL Workbench](https://www.mysql.com/products/workbench/)
+- [SQLYog](https://www.webyog.com/product/sqlyog) (Paid)
+
+#### SSH Clients:
+- DigitalOcean’s Console
+- [PuTTY](https://www.chiark.greenend.org.uk/~sgtatham/putty/)
+- [Terminus](https://termius.com/) (Paid)
+
+#### FTP Clients:
+- [Filezilla](https://filezilla-project.org/)
+- [Terminus](https://termius.com/) (Paid)
+
+#### File Editors:
+- [Notepad++](https://notepad-plus-plus.org/)
```

Comment:
maybe we can add Visual Studio Code to the list here, since it's multi-platform and completely free and open source:

https://code.visualstudio.com/